template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Areas/IdentityService/Views/IdentityService/LoggedOut.cshtml

```diff
@@ -0,0 +1,5 @@
+@{
+    ViewData["Title"] = "Sign Out";
```

Comment:
Yeah, I think we should clean this up if possible. My preference is sign-in and sign-out. I'll take a look and see how that shakes out. Let me know if anyone strongly disagrees.